e2e/cluster/cluster_e2e_test.go

```diff
@@ -0,0 +1,138 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"os/exec"
+	"testing"
+	"time"
+
+	"github.com/grafana/xk6-disruptor/pkg/testutils/cluster"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func Test_DefaultConfig(t *testing.T) {
+	// create cluster with default configuration
+	config, err := cluster.NewClusterConfig(
+		"e2e-default-cluster",
+		cluster.ClusterOptions{
+			Wait: time.Second * 60,
+		},
+	)
+	if err != nil {
+		t.Errorf("failed creating cluster configuration: %v", err)
+		return
+	}
+
+	cluster, err := config.Create()
+	if err != nil {
+		t.Errorf("failed to create cluster: %v", err)
+		return
+	}
+
+	// delete cluster
+	cluster.Delete()
+}
+
+func getKubernetesClient(kubeconfig string) (kubernetes.Interface, error) {
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
+	if err != nil {
+		return nil, err
+	}
+
+	client, err := kubernetes.NewForConfig(config)
+	if err != nil {
+		return nil, err
+	}
+
+	return client, nil
+}
+
+// buildBusyboxPod returns a pod specification for running Busybox from a local image
+func buildBusyboxPod() *corev1.Pod {
+	return &corev1.Pod{
```

Comment:
In the HTTP tests, we use strings with the YAML inside. 

It would be nice to use the same approach in both places. Unsure about which one is better, though.